src/main/java/de/tum/in/www1/artemis/service/exam/ExamService.java

```diff
@@ -200,6 +200,9 @@ public ExamScoresDTO calculateExamScores(Long examId) {
         // Adding exam information to DTO
         ExamScoresDTO scores = new ExamScoresDTO(exam.getId(), exam.getTitle(), exam.getMaxPoints());
 
+        // setting multiplicity of correction rounds
+        scores.hasSecondCorrectionAndStarted = false;
```

Comment:
We can set more than two correction rounds in exams.
Having this specific variable for the second round could be a problem.

Reply:
This would be new to me. When I create an exam i can select either one or two correction rounds. Are you sure?

Reply:
![Bildschirmfoto 2021-06-15 um 09 36 29](https://user-images.githubusercontent.com/38322605/122012236-3cf2f300-cdbd-11eb-9ce8-94aa2bc30bfb.png)
(Taken on the current production server)

Reply:
If you change the focus and click on any other place on the page, the correction round input field should appear read and therefore invalid:
<img width="701" alt="Screenshot 2021-06-16 at 14 30 26" src="https://user-images.githubusercontent.com/44401560/122219144-6d1abe80-ceaf-11eb-8c49-5ae22559fdd2.png">

If you can actually create an exam with more than 2 correction rounds, this could be a potential bug, since we only support two correction rounds, or am I wrong @SimonEntholzer @nicolasruscher ?